TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -255,7 +255,8 @@ private static String formatAuthArgFlagsAsString() {
                 .collect(Collectors.joining(", "));
     }
 
-    private static IAuthTransformerFactory buildAuthTransformerFactory(Parameters params) {
+    private static IAuthTransformerFactory buildAuthTransformerFactory(Parameters params)
+        throws ExecutionException, InterruptedException {
```

Comment:
I can't imagine a case where I'd need to make these factories asynchronously.  I could see needing to create the AuthTransformers themselves asynchronously, or signing the requests - but this seems like it would always be part of bootstrapping at the top level.

Reply:
I guess I am inclined to agree with you. Initially when I set this up I thought this might be used in the Netty pipeline so having async would be useful, but for our current use case I really only see synchronous and supporting the off case where someone's secrets has changed in the middle of a migration doesn't seem immediate. I've refactored to use a synchronous client now

Reply:
I also included the fix for the failing test case since I don't plan to merge this to the release branch